tests/lib/shared/option-utils.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @fileoverview Tests for string utils.
+ * @author Josh Goldberg
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const assert = require("node:assert");
+
+const {
+    containsDifferentProperty
+} = require("../../../lib/shared/option-utils");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+describe("containsDifferentProperty", () => {
+    [
+        [null, null, false],
+        [null, {}, true],
+        [{}, null, true],
+        [null, null, false],
+        [null, void 0, true],
+        [void 0, null, true],
+        [void 0, {}, true],
+        [{}, void 0, true],
+        [0, 0, false],
+        [0, 1, true],
+        [1, 0, true],
```

Comment:
```suggestion
        [1, 0, true],
        [[], {}, true],
        [{}, [], true],
```

Test cases for the above change (the second one was failing).